sotodlib/tod_ops/jumps.py

```diff
@@ -457,7 +463,7 @@ def twopi_jumps(
         atol = 3 * std_est(signal.astype(float))
         np.clip(atol, 1e-8, 1e-2)
 
-    _signal = _filter(signal, gaussian_width, tv_weight)
+    _signal = _filter(signal, 3, gaussian_width, tv_weight)
```

Comment:
what exactly is this hardcoded in `medfilt = 3` needed for?

Reply:
That kills glitches in data without really effecting much else. It may not actually be needed but I was imagining a scenario where there is a glitch sufficiently close to to a jump such that it confuses things.

Reply:
Maybe what I really want is to just make that a kwarg with a sane default so that it can be turned off if desired.